airbyte-ci/connectors/metadata_service/orchestrator/orchestrator/__init__.py

```diff
@@ -93,9 +96,17 @@
     ),
 }
 
+CONNECTOR_METRICS_RESOURCE_TREE = {
+    **GCS_RESOURCE_TREE,
+    "latest_metrics_gcs_blob": gcs_directory_blobs.configured(
+        {"gcs_bucket": ANALYTICS_BUCKET, "prefix": ANALYTICS_FOLDER, "match_regex": f".*.jsonl$", "only_one": True, "sort_key": "name", "reverse_sort": True}
```

Comment:
❓ I'm not aware of the filename templating you decided to use, but from reading this code I assume ordering matters to define which file contains the latest metrics. Shall we make the `match_regex` stricter to match `jsonl` files with the correct prefixes?
It could be helpful to gain control if we eventually decide to change the filename templating logic.

Reply:
So heres a path to the file `ab-analytics-connector-metrics/data/connector_quality_metrics/2024_05_26_1716765525385_0.jsonl`

The reason I has a loose regex is you can see theres not much else worth matching on as they are just timestamps.